smart-client-jersey/src/main/java/com/emc/rest/smart/jersey/SmartFilter.java

```diff
@@ -75,9 +76,11 @@ public ClientResponse handle(ClientRequest request) throws ClientHandlerExceptio
             return response;
         } catch (RuntimeException e) {
 
-            // capture requests stats (error)
-            host.callComplete(true);
-            host.connectionClosed();
+            if (e instanceof SmartException && !((SmartException)e).getErrorType().equals(SmartException.ErrorType.Client)) {
```

Comment:
Is it better to add http code to the new exception class, thus error type can be decided by httpCode so that you don't have to make code change in object-client when throwing S3Exception?